examples/ios/swift/Migration/README.md

```diff
@@ -0,0 +1,28 @@
+# Migration Examples
+
+The Migration project shows several examples of migrations and migration blocks.
+
+The purpose of this example is to provide a more in depth view of certain problems and pitfalls users might face when migrations get more complex and the number of versions increases.
+
+## How to use the example
+
+You can build and run the project since this is the default state (see `CREATE_EXAMPLES` pre-compiler flag in `AppDelegate`).
+There is nothing much so see in terms of unit tests, console log or in the app itself. If there is no error the migration was successful.
```

Comment:
You should `assert` objects to see if they contain the correct data. The 'no news is good news' approach here is flaky.